packages/medusa-plugin-sendgrid/src/services/sendgrid.js

```diff
@@ -221,10 +222,14 @@ class SendGridService extends NotificationService {
     let templateId = this.getTemplateId(event)
 
     if (!templateId) {
-      return false
+      throw new MedusaError(MedusaError.Types.INVALID_DATA, "Sendgrid service: No template was set for this event")
```

Comment:
thought(non-blocking): Should we also log the event?

Reply:
Does that string get interpolated?  Can I do:

```
`Sendgrid service: No template was set for this ${event}`
```

Reply:
yes you can do that 👍